src/main/java/org/elasticsearch/search/internal/SearchContext.java

```diff
@@ -163,6 +164,8 @@ public final boolean nowInMillisUsed() {
 
     public abstract InnerHitsContext innerHits();
 
+    public abstract Set<FieldMapper<?>> aliasFields();
```

Comment:
good point, I found this naming here tricky, includeFields is better, it just described what it does.